Source/WebCore/page/IntelligenceTextEffectsSupport.cpp

```diff
@@ -85,6 +85,9 @@ std::optional<TextIndicatorData> textPreviewDataForRange(Document&, const Simple
         TextIndicatorOption::SkipReplacedContent,
         TextIndicatorOption::RespectTextColor,
         TextIndicatorOption::DoNotClipToVisibleRect,
+#if PLATFORM(VISION)
+        TextIndicatorOption::SnapshotContentAt3xBaseScale,
```

Comment:
I could use suggestions for a better name for this plus the corresponding snapshot option

Reply:
yeah, bikeshedding this is pretty much the only thing I can see with this patch otherwise it looks sensible